test/playback.test.js

```diff
@@ -57,9 +54,11 @@ QUnit.module('Playback', {
   }
 });
 
-QUnit.test('Advanced Bip Bop', function(assert) {
+QUnit.test('Advanced Bip Bop default speed', function(assert) {
   let done = assert.async();
 
+  this.player.defaultPlaybackRate(1);
```

Comment:
fyi, this can also be just `playbackRate`. I don't think it makes a huge difference.

Reply:
I tried that, but it seems like if we try to set a playbackRate before we have a source that is ready, we reset to the `defaultPlaybackRate` upon playback.

Reply:
Ah, makes sense. Carry on then :)